demos/interactive_face_detection_demo/cpp_gapi/main.cpp

```diff
@@ -456,10 +456,10 @@ int main(int argc, char *argv[]) {
                 throw std::invalid_argument(msg.str());
             }
 
-            timer.start("total");
+            auto startTime = std::chrono::steady_clock::now();
```

Comment:
Collecting `startTime` like this still suggests that the latency is collected, just not reported. Try providing a placeholder, or update `update()` func.

Reply:
Changed `startTime` to a placeholder in `update() `function, but it requires time stamp of the first start for proper FPS measurement, so one time before cycle there would be time measurment.

Reply:
But in this implementation, due to `update()` function call at the start of cycle frame counter has +1 in comparison with real frame's number.

Reply:
Start timer at where `update()` called outside of the loop, and pass the time point to the first `update()` called in the loop.

Reply:
Done, https://github.com/openvinotoolkit/open_model_zoo/pull/2647/commits/ab508a935851da8d9cefa687228e36e44952e1ac